RentACarTests/ControllersTests/ClientControllerTests.cs

```diff
@@ -0,0 +1,254 @@
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using RentACarWebAPI.Controllers;
+using RentACarWebAPI.DTOs;
+using RentACarWebAPI.Interfaces.Services;
+using RentACarWebAPI.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace RentACarTests.ControllersTests
+{
+    public class ClientControllerTests
+    {
+        #region GetAllAsync
+        [Fact]
+        public async Task GetAllAsync_ReturnsClientsDto()
+        {
+            //arrange
+            var clientOneName = "Pedro";
+
+            var client1 = new Client() { Id = 1, Name = clientOneName, Dni = "13131311" };
+            var client2 = new Client() { Id = 3, Name = "Juanse", Dni = "13131313" };
+            var client3 = new Client() { Id = 7, Name = "Sofia", Dni = "89444666" };
```

Comment:
what's the difference between client1 and client2? why just not create all clients in the same way and assert they were all returned?

Reply:
When i started to code this, i had checked if the name of client1 was the same as client1 of the returned list (i was a bit confused in the beginning), so then i changed it and left the same clients. The reason is just anecdotal and I have no excuses! I will change it, thanks!

Reply:
no problem, just try to look for consistency and not to overengineer everything (pretty difficult, I know, but not impossible, just ask for guidance)